File: app/src/main/java/org/michaelbel/tjgram/data/entity/AdditionalData.java
package org.michaelbel.tjgram.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class AdditionalData implements Serializable {
    @Expose @SerializedName("duration") public Double duration;
    @Expose @SerializedName("hasAudio") public boolean isHasAudio;
    @Expose @SerializedName("size") public Integer size;
    @Expose @SerializedName("type") public String type;
    @Expose @SerializedName("url") public String url;
    @Expose @SerializedName("uuid") public String uuid;
}

File: app/src/main/java/org/michaelbel/tjgram/data/entity/AdditionalData.kt
package org.michaelbel.tjgram.data.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

data class AdditionalData (
    @Expose @SerializedName("duration") var duration: Double,
    @Expose @SerializedName("hasAudio") var isHasAudio: Boolean,
    @Expose @SerializedName("size") var size: Int,
    @Expose @SerializedName("type") var type: String,
    @Expose @SerializedName("url") var url: String,
    @Expose @SerializedName("uuid") var uuid: String
): Serializable